app/__init__.py

```diff
@@ -18,6 +19,17 @@
 content_loader = ContentLoader('app/content')
 
 
+def _try_load_manifest(application, data, question_set, manifest):
+    try:
+        content_loader.load_manifest(data['slug'], question_set, manifest)
+
+    except ContentNotFoundError:
+        # Not all frameworks have this, so no need to panic (e.g. G-Cloud 4, G-Cloud 5, G-Cloud-6)
```

Comment:
I don't think the comment quite works here though, because it could be anything it's trying to load, even something that all frameworks should have...